prime-router/src/main/kotlin/azure/ActionHistory.kt

```diff
@@ -574,20 +619,34 @@ class ActionHistory {
         organization: Organization,
         sendingAt: OffsetDateTime?,
         countToPrint: Int,
-        reportOptions: ReportFunction.Options
+        reportOptions: ReportFunction.Options,
+        reportId: ReportId? = null
     ) {
         jsonGen.writeStartObject()
         // jsonGen.writeStringField("id", reportFile.reportId.toString())   // TMI?
         jsonGen.writeStringField("organization", organization.description)
         jsonGen.writeStringField("organization_id", orgReceiver.organizationName)
         jsonGen.writeStringField("service", orgReceiver.name)
 
+        reportId?.let {
+            if (filteredReportRows.contains(it)) {
+                jsonGen.writeArrayFieldStart("filteredReportRows")
+                filteredReportRows.getValue(it).forEach {
+                    jsonGen.writeString(it.toString())
+                }
+                jsonGen.writeEndArray()
+            }
+        }
+
         jsonGen.writeStringField(
             "sending_at",
             when {
                 reportOptions == ReportFunction.Options.SkipSend -> {
                     "never - skipSend specified"
                 }
+                countToPrint == 0 -> {
```

Comment:
Is there a doc that will need to be changed due to these and other response changes?  I don't think I have seen one.

Reply:
looks like at least the [openapi.yml](https://github.com/CDCgov/prime-reportstream/blob/master/prime-router/docs/openapi.yml#L537-L554) needs updating